### DISCUSSION TITLE
Minimal build and extra functions from the standard library

### DISCUSSION THREAD

**Discussion by vitoyucepi** [savonet/liquidsoap#3264] [Q&A]
Why are some functions, that will work perfectly with the minimal build, marked as extra? 
For example, `track.audio.compress.exponential` from main and `compress.exponential` from extra.

**Comment by smimram** [comment]
The distinction is rather between very useful or not functions in order to try reduce the clutter in the doc. But I agree that the line is sometimes unclear...

**Reply by vitoyucepi** [reply]
#3143

**Reply by toots** [reply]
It's also for prioritizing bug fixes. Stuff in the core library, we do want them to work at all time. Stuff in the extra library are usually more complex and advanced functions with more niche use. An bugfix in them might not trigger a short term bugfix release.

Eventually, we also want to consider releasing the standard library/extra libraries independently.
